addons/base_import/models/base_import.py

```diff
@@ -1459,9 +1464,39 @@ def execute_import(self, fields, columns, options, dryrun=False):
         # convert load's internal nextrow to the imported file's
         if import_result['nextrow']: # don't update if nextrow = 0 (= no nextrow)
             import_result['nextrow'] += skip
+        if additional_filenames:
+            import_result['additional_filenames'] = additional_filenames
 
         return import_result
 
+    def _extract_additional_filenames(self, import_fields, data, model=False, prefix='', additional_filenames=False):
+        model = model or self.res_model
+        additional_filenames = additional_filenames or {}
```

Comment:
Tou can use a defaultdict to avoid manually doing `additional_filenames[name] = []`
```suggestion
        additional_filenames = additional_filenames or defaultdict(list)
```